CONTRIBUTING.md

```diff
@@ -15,6 +19,32 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Data file management
+
+### Setup directory
+
+Each module should contain a `setup` directory that includes instructions and scripts to download and prepare any input data files used by the notebooks in the module.
+
+### Linking shared files 
+
+When trainings are run from the RStudio server (rstudio.ccdatalab.org), we store large input data files in the `/shared/data/training-modules` directory.
+The organization within this directory should mirror the arrangement of the repository, so that we can easily link files and folders from this shared directory to mirrored paths within a clone of this repository.
```

Comment:
What's missing for me in here (but I would say it is _implicit_) is what I need to do if I have new files, i.e., when I am working on pathway analysis changes, the single point of truth is `/shared` so that's where I should put them during development and then I add them to `scripts/link-data.sh`.

Reply:
Yes, good point, will add this.